src/types.ts

```diff
@@ -201,9 +229,65 @@ export enum ActivityType {
 export interface Activity {
   address: string
   metadata: string
-  auctionHouse: string
+  auctionHouse: AuctionHouse
   price: BN
   createdAt: string
   wallets: string[]
   activityType: string
+  nft: Nft
+}
+
+export interface TwitterProfile {
+  handle: string
+  profileImageUrl: string
```

Comment:
<img width="620" alt="Screen Shot 2022-06-06 at 11 42 41 AM" src="https://user-images.githubusercontent.com/2388118/172136946-098f6ea2-43ad-40a1-8bdb-0cebf1c73ff4.png">

profileImageUrl been deprecated can you make it optional and replace with the high and low res variants?

Make all the fields optional except the handle.